website/includes/donor-contact/fa-body.html

```diff
@@ -0,0 +1,93 @@
+          <div class="inputWrapper"><input type="text" id="tfa_1" name="tfa_1" value="" title="Contact Name" class="" required>
+          </div>
+        </div>
+        <div class="oneField field-container-D    " id="tfa_10-D">
+          <label id="tfa_10-L" class="label preField " for="tfa_10">{{ _('Email') }}</label>
+          <div class="inputWrapper"><input type="text" id="tfa_10" name="tfa_10" value="" title="Email" class="" required></div>
+        </div>
+        <div class="oneField field-container-D    " id="tfa_72-D">
+          <label id="tfa_72-L" class="label preField " for="tfa_72">{{ _('Language') }}</label>
+          <div class="inputWrapper"><select id="tfa_72" name="tfa_72" title="Language" class="">
+            <option value="">{{ _('Please select...') }}</option>
+            <option value="tfa_73" id="tfa_73" class="">{{ _('Chinese Simplified') }}</option>
+            <option value="tfa_74" id="tfa_74" class="">{{ _('Czech') }}</option>
+            <option value="tfa_75" id="tfa_75" class="">{{ _('Danish') }}</option>
+            <option value="tfa_76" id="tfa_76" class="">{{ _('Dutch') }}</option>
+            <option value="tfa_77" id="tfa_77" class="">{{ _('English') }}</option>
+            <option value="tfa_78" id="tfa_78" class="">{{ _('French') }}</option>
+            <option value="tfa_79" id="tfa_79" class="">{{ _('German') }}</option>
+            <option value="tfa_80" id="tfa_80" class="">{{ _('Italian') }}</option>
+            <option value="tfa_81" id="tfa_81" class="">{{ _('Japanese') }}</option>
+            <option value="tfa_82" id="tfa_82" class="">{{ _('Other') }}</option>
+            <option value="tfa_83" id="tfa_83" class="">{{ _('Polish') }}</option>
+            <option value="tfa_84" id="tfa_84" class="">{{ _('Portuguese') }}</option>
+            <option value="tfa_85" id="tfa_85" class="">{{ _('Russian') }}</option>
+            <option value="tfa_86" id="tfa_86" class="">{{ _('Spanish') }}</option>
+          </select></div>
+        </div>
+        <div class="oneField field-container-D    " id="tfa_95-D">
+          <label id="tfa_95-L" class="label preField " for="tfa_95">{{ _('Subject') }}</label>
+          <div class="inputWrapper"><input type="text" id="tfa_95" name="tfa_95" value="" title="Subject" class="" required>
+          </div>
+        </div>
+        <div id="tfa_183" class="section inline group">
+          <div class="oneField field-container-D    " id="tfa_163-D">
+            <label id="tfa_163-L" class="label preField " for="tfa_163">{{ _('Description') }}</label>
+            <div class="inputWrapper"><textarea id="tfa_163" name="tfa_163" title="Description" class="" required></textarea>
+            </div>
+          </div>
+        </div>
+        <div class="htmlSection" id="tfa_182">
+          <div class="htmlContent" id="tfa_182-HTML">{{ _('Mozilla will only use your submitted information for purposes of
```

Comment:
This is new text, do we need to change this to MZLA/Thunderbird @Sancus ?

Reply:
Given this data *is* going to Mozilla Foundation staff, and the privacy policy is also on their site, we should probably keep that as-is.